api/src/main.js

```diff
@@ -44,5 +47,68 @@ if (import.meta.env.PROD) {
   });
 }
 
+// ---------------------------------------------------------------------------------------------------------------------
```

Comment:
Please delete useless comments

Reply:
I don't see any emergency here, it is just to seperate clean code and no clean code